### ISSUE TITLE
Test code full of bugs

### ISSUE THREAD

**Issue by davidsonic** [charliememory/Pose-Guided-Person-Image-Generation#1]
Even after lots of debugging, the downloaded pre-trained model cannot be loaded with the following error: 

DataLossError (see above for traceback): Unable to open table file ./pretrain_models/PG2_model_D│face_morpher                Sketch-Photo-Conversion-using-Deep-CNN
F/model.ckpt-0.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your f│FastPhotoStyle              SketchToFace
ile is in a different file format and you need to use a different restore operator?

**Comment by charliememory** [comment]
Sorry for the late update. Bugs in the testing code are fixed now.
